pkgs/applications/version-management/git-bars/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, lib, fetchFromGitHub, python3Packages, git }:
+
+python3Packages.buildPythonApplication {
+  pname = "git-bars";
+  version = "unstable-2023-08-08";
+
+  src = fetchFromGitHub {
+    owner = "knadh";
+    repo = "git-bars";
+    rev = "f15fbc15345d9ef021e5a9b278e352bb532dcee8";
+    hash = "sha256-jHP6LqhUQv6hh97tSXAdOruWdtp2FXM6ANlpWoA+fHQ=";
+  };
+
+  propagatedBuildInputs = [
+    git
+    python3Packages.setuptools
+  ];
+
+  meta = with lib; {
+    homepage = "https://github.com/knadh/git-bars";
+    description = "A utility for visualising git commit activity as bars on the terminal";
+    license = licenses.unfree; # no license in repository
```

```suggestion
    license = licenses.mit;
```

It was updated yesterday. What about update this expression too?